### PR TITLE
nonbinary.tech post

### PR DESCRIPTION
**Pull request by WheresAlice**

### CODE REVIEW

File: _posts/2017-07-30-nonbinary-tech.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Damien Senger gave a very personal talk about **Building a New Life After Burnout**.  They reminded us the importance of saying no, taking breaks, and making deliberate choices to take care of ourselves.  Initial stages of burnout can show itself as us being less performant, poor memory, working more than the regular hours, fatigue, unstable mood, loneliness and exclusion, and risk behaviours.  It's important to take the time to do something about this as early as you notice this, even if it is just moving to another team.
```
Comment (brabster) [inline _posts/2017-07-30-nonbinary-tech.md]: "...can show itself as us being less performant, poor memory, ..." doesn't seem to read well to me, something like "reduced performance, poor memory" perhaps

"It's important to take the time to do something about this as early as you notice this, even if it is just moving to another team." - use of "this" twice in quick succession, perhaps "about this as early as you notice it"?

Comment (WheresAlice) [inline _posts/2017-07-30-nonbinary-tech.md]: updated

Comment (mitchellrj) [inline _posts/2017-07-30-nonbinary-tech.md]: > Initial stages of burnout can show itself as us being less performant, poor memory, working more than the regular hours, fatigue, unstable mood, loneliness and exclusion, and risk behaviours.

The wording here is a bit strained. I'd rephrase as:

> Symptoms of the initial stages of burnout include decreased performance; poor memory; working longer hours; fatigue; mood swings; loneliness and exclusion and; increased high risk behaviours.

File: _posts/2017-07-30-nonbinary-tech.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Olu shared a lot of great resources they have used through the beginning of their career in **Beyond 1s and 0s: inclusive spaces in binary places**.  [Diversity Tickets](https://diversitytickets.org) is a great place to find conferences that have ticket and travel grants for minority groups.  [WeAllJS](https://wealljs.org) is an inclusive Slack community for JavaScript developers.  The [LGBTQ in Technology](http://lgbtq.technology) Slack organisation is a space for LGBTQ people in technology to chat and support each other.  The [Women in Technology Chat](http://witchat.github.io) is a Slack organisation for women who work in technology.  Joining these communities helps with a sense of belonging, feeling welcomed and being understood which can be lacking elsewhere.  And remember to read and use the codes of conduct.
```
Comment (mitchellrj) [inline _posts/2017-07-30-nonbinary-tech.md]: The "And" in the last sentence is redundant I think.

Comment (WheresAlice) [inline _posts/2017-07-30-nonbinary-tech.md]: I think the "And" in the last sentence is important, as it reinforces the idea that all of the previous things are good on their own but the codes of conduct which sit on top of them are so important.

File: _posts/2017-07-30-nonbinary-tech.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+**Diversity by Default in Design** was given by a designer about their efforts to show diversity in their artwork, and the battles with companies to actually get that artwork used.  Eriol talked about how "edge-case" has become a word to discourage time being spent on making products accessible to marginalized groups and how making products friendly is universal, not just for the dominant user group.  Stock images for minority groups can throw up some worrying results, illiciting negative stereotypes, and it is important to remember that just because a stock image exists doesn't mean it is acceptable to use.  If you are in a position to represent your product users through photography, use photographs of actual users; and if those photographs don't show diversity then maybe you have bigger problems to address first.
```
Comment (peterohanra) [inline _posts/2017-07-30-nonbinary-tech.md]: hi richard @mitchellrj  . how are those google play reviews doing. seems you like live casino. i am quite impressed by your skills and information security. everyone at big g00gles is impressed too . are those reviews still there. what kind of servers is in your data center? oh ok i found some.

File: _posts/2017-07-30-nonbinary-tech.md (hunk 2 above)
Comment (peterohanra) [inline _posts/2017-07-30-nonbinary-tech.md]: @mitchellrj  richard did you get email about npm vulnerabilities type squatting today.  does any your red hat have these repo in npm. i can check for you now. but enough for now richard. do the right thing. if you know of any other people in your organisation who are good at being honest then lets see what honest they can do too. there is edit or delete button in store that is fun to play.

File: _posts/2017-07-30-nonbinary-tech.md (hunk 1 above)
Comment (peterohanra) [inline _posts/2017-07-30-nonbinary-tech.md]: @mitchellrj but richard 26mb?!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!??!?!?!?!?!!? did you rotate the phone?? did it crash? wat? what is info sec. do you know basic. good bye richar. lets not meet again.

**Pull request merged**